{"uid":"8723fb6f7e58c013","name":"test_05_alerts_handling[edge]","fullName":"selenium_demoblaze_framework.tests.test_demoblaze_e2e.TestDemoBlazeE2E#test_05_alerts_handling","historyId":"c0e9249d4f102437f6119fa11051c0e1","time":{"start":1761242295935,"stop":1761242295935,"duration":0},"description":"Test handling of alerts.","descriptionHtml":"<p>Test handling of alerts.</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.SessionNotCreatedException: Message: session not created\nfrom disconnected: unable to connect to renderer\nStacktrace:\n#0 0x562cd54aacce <unknown>\n#1 0x562cd4f34656 <unknown>\n#2 0x562cd4f74ada <unknown>\n#3 0x562cd4f707e9 <unknown>\n#4 0x562cd4f69e6d <unknown>\n#5 0x562cd4fb9663 <unknown>\n#6 0x562cd4fb8de6 <unknown>\n#7 0x562cd4faa5d3 <unknown>\n#8 0x562cd4f78228 <unknown>\n#9 0x562cd4f78e11 <unknown>\n#10 0x562cd547a16e <unknown>\n#11 0x562cd547d2cb <unknown>\n#12 0x562cd547cd2b <unknown>\n#13 0x562cd547d785 <unknown>\n#14 0x562cd54674e3 <unknown>\n#15 0x562cd547db52 <unknown>\n#16 0x562cd5451242 <unknown>\n#17 0x562cd5499078 <unknown>\n#18 0x562cd549929c <unknown>\n#19 0x562cd54a9b59 <unknown>\n#20 0x7f278dc9caa4 <unknown>\n#21 0x7f278dd29c6c <unknown>","statusTrace":"selenium_demoblaze_framework/config/browser_config.py:182: in get_driver\n    driver = webdriver.Edge(service=service, options=self.get_edge_options())\n/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/edge/webdriver.py:45: in __init__\n    super().__init__(\n/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/chromium/webdriver.py:51: in __init__\n    self.service.path = DriverFinder.get_path(self.service, options)\n/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/common/driver_finder.py:44: in get_path\n    raise NoSuchDriverException(f\"Unable to locate or obtain driver for {options.capabilities['browserName']}\")\nE   selenium.common.exceptions.NoSuchDriverException: Message: Unable to locate or obtain driver for MicrosoftEdge; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location\n\nDuring handling of the above exception, another exception occurred:\nselenium_demoblaze_framework/tests/test_demoblaze_e2e.py:1147: in setup_and_teardown\n    self.driver = self.browser_config.get_driver(browser_name)\nselenium_demoblaze_framework/config/browser_config.py:188: in get_driver\n    driver = webdriver.Edge(service=service, options=self.get_edge_options())\n/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/edge/webdriver.py:45: in __init__\n    super().__init__(\n/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/chromium/webdriver.py:56: in __init__\n    super().__init__(\n/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py:207: in __init__\n    self.start_session(capabilities)\n/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py:291: in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py:346: in execute\n    self.error_handler.check_response(response)\n/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py:229: in check_response\n    raise exception_class(message, screen, stacktrace)\nE   selenium.common.exceptions.SessionNotCreatedException: Message: session not created\nE   from disconnected: unable to connect to renderer\nE   Stacktrace:\nE   #0 0x562cd54aacce <unknown>\nE   #1 0x562cd4f34656 <unknown>\nE   #2 0x562cd4f74ada <unknown>\nE   #3 0x562cd4f707e9 <unknown>\nE   #4 0x562cd4f69e6d <unknown>\nE   #5 0x562cd4fb9663 <unknown>\nE   #6 0x562cd4fb8de6 <unknown>\nE   #7 0x562cd4faa5d3 <unknown>\nE   #8 0x562cd4f78228 <unknown>\nE   #9 0x562cd4f78e11 <unknown>\nE   #10 0x562cd547a16e <unknown>\nE   #11 0x562cd547d2cb <unknown>\nE   #12 0x562cd547cd2b <unknown>\nE   #13 0x562cd547d785 <unknown>\nE   #14 0x562cd54674e3 <unknown>\nE   #15 0x562cd547db52 <unknown>\nE   #16 0x562cd5451242 <unknown>\nE   #17 0x562cd5499078 <unknown>\nE   #18 0x562cd549929c <unknown>\nE   #19 0x562cd54a9b59 <unknown>\nE   #20 0x7f278dc9caa4 <unknown>\nE   #21 0x7f278dd29c6c <unknown>","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1761242291677,"stop":1761242291841,"duration":164},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"session_browser","time":{"start":1761242291841,"stop":1761242291842,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"set_browser_env","time":{"start":1761242291842,"stop":1761242291842,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"setup_and_teardown","time":{"start":1761242295936,"stop":1761242297043,"duration":1107},"status":"broken","statusMessage":"selenium.common.exceptions.SessionNotCreatedException: Message: session not created\nfrom disconnected: unable to connect to renderer\nStacktrace:\n#0 0x562cd54aacce <unknown>\n#1 0x562cd4f34656 <unknown>\n#2 0x562cd4f74ada <unknown>\n#3 0x562cd4f707e9 <unknown>\n#4 0x562cd4f69e6d <unknown>\n#5 0x562cd4fb9663 <unknown>\n#6 0x562cd4fb8de6 <unknown>\n#7 0x562cd4faa5d3 <unknown>\n#8 0x562cd4f78228 <unknown>\n#9 0x562cd4f78e11 <unknown>\n#10 0x562cd547a16e <unknown>\n#11 0x562cd547d2cb <unknown>\n#12 0x562cd547cd2b <unknown>\n#13 0x562cd547d785 <unknown>\n#14 0x562cd54674e3 <unknown>\n#15 0x562cd547db52 <unknown>\n#16 0x562cd5451242 <unknown>\n#17 0x562cd5499078 <unknown>\n#18 0x562cd549929c <unknown>\n#19 0x562cd54a9b59 <unknown>\n#20 0x7f278dc9caa4 <unknown>\n#21 0x7f278dd29c6c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n  File \"/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/pluggy/_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n  File \"/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/pluggy/_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n  File \"/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/selenium-demoblaze-operations/selenium-demoblaze-operations/selenium_demoblaze_framework/tests/test_demoblaze_e2e.py\", line 1147, in setup_and_teardown\n    self.driver = self.browser_config.get_driver(browser_name)\n  File \"/home/runner/work/selenium-demoblaze-operations/selenium-demoblaze-operations/selenium_demoblaze_framework/config/browser_config.py\", line 188, in get_driver\n    driver = webdriver.Edge(service=service, options=self.get_edge_options())\n  File \"/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/edge/webdriver.py\", line 45, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/chromium/webdriver.py\", line 56, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 207, in __init__\n    self.start_session(capabilities)\n  File \"/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 291, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n  File \"/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true}],"testStage":{"description":"Test handling of alerts.","status":"broken","statusMessage":"selenium.common.exceptions.SessionNotCreatedException: Message: session not created\nfrom disconnected: unable to connect to renderer\nStacktrace:\n#0 0x562cd54aacce <unknown>\n#1 0x562cd4f34656 <unknown>\n#2 0x562cd4f74ada <unknown>\n#3 0x562cd4f707e9 <unknown>\n#4 0x562cd4f69e6d <unknown>\n#5 0x562cd4fb9663 <unknown>\n#6 0x562cd4fb8de6 <unknown>\n#7 0x562cd4faa5d3 <unknown>\n#8 0x562cd4f78228 <unknown>\n#9 0x562cd4f78e11 <unknown>\n#10 0x562cd547a16e <unknown>\n#11 0x562cd547d2cb <unknown>\n#12 0x562cd547cd2b <unknown>\n#13 0x562cd547d785 <unknown>\n#14 0x562cd54674e3 <unknown>\n#15 0x562cd547db52 <unknown>\n#16 0x562cd5451242 <unknown>\n#17 0x562cd5499078 <unknown>\n#18 0x562cd549929c <unknown>\n#19 0x562cd54a9b59 <unknown>\n#20 0x7f278dc9caa4 <unknown>\n#21 0x7f278dd29c6c <unknown>","statusTrace":"selenium_demoblaze_framework/config/browser_config.py:182: in get_driver\n    driver = webdriver.Edge(service=service, options=self.get_edge_options())\n/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/edge/webdriver.py:45: in __init__\n    super().__init__(\n/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/chromium/webdriver.py:51: in __init__\n    self.service.path = DriverFinder.get_path(self.service, options)\n/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/common/driver_finder.py:44: in get_path\n    raise NoSuchDriverException(f\"Unable to locate or obtain driver for {options.capabilities['browserName']}\")\nE   selenium.common.exceptions.NoSuchDriverException: Message: Unable to locate or obtain driver for MicrosoftEdge; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location\n\nDuring handling of the above exception, another exception occurred:\nselenium_demoblaze_framework/tests/test_demoblaze_e2e.py:1147: in setup_and_teardown\n    self.driver = self.browser_config.get_driver(browser_name)\nselenium_demoblaze_framework/config/browser_config.py:188: in get_driver\n    driver = webdriver.Edge(service=service, options=self.get_edge_options())\n/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/edge/webdriver.py:45: in __init__\n    super().__init__(\n/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/chromium/webdriver.py:56: in __init__\n    super().__init__(\n/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py:207: in __init__\n    self.start_session(capabilities)\n/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py:291: in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py:346: in execute\n    self.error_handler.check_response(response)\n/opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py:229: in check_response\n    raise exception_class(message, screen, stacktrace)\nE   selenium.common.exceptions.SessionNotCreatedException: Message: session not created\nE   from disconnected: unable to connect to renderer\nE   Stacktrace:\nE   #0 0x562cd54aacce <unknown>\nE   #1 0x562cd4f34656 <unknown>\nE   #2 0x562cd4f74ada <unknown>\nE   #3 0x562cd4f707e9 <unknown>\nE   #4 0x562cd4f69e6d <unknown>\nE   #5 0x562cd4fb9663 <unknown>\nE   #6 0x562cd4fb8de6 <unknown>\nE   #7 0x562cd4faa5d3 <unknown>\nE   #8 0x562cd4f78228 <unknown>\nE   #9 0x562cd4f78e11 <unknown>\nE   #10 0x562cd547a16e <unknown>\nE   #11 0x562cd547d2cb <unknown>\nE   #12 0x562cd547cd2b <unknown>\nE   #13 0x562cd547d785 <unknown>\nE   #14 0x562cd54674e3 <unknown>\nE   #15 0x562cd547db52 <unknown>\nE   #16 0x562cd5451242 <unknown>\nE   #17 0x562cd5499078 <unknown>\nE   #18 0x562cd549929c <unknown>\nE   #19 0x562cd54a9b59 <unknown>\nE   #20 0x7f278dc9caa4 <unknown>\nE   #21 0x7f278dd29c6c <unknown>","steps":[],"attachments":[{"uid":"944662375a819c79","name":"log","source":"944662375a819c79.txt","type":"text/plain","size":5162},{"uid":"d4c7047aa1e192ba","name":"stdout","source":"d4c7047aa1e192ba.txt","type":"text/plain","size":285},{"uid":"a0117ae40fcd190","name":"stderr","source":"a0117ae40fcd190.txt","type":"text/plain","size":225}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":3,"hasContent":true},"afterStages":[{"name":"set_browser_env::0","time":{"start":1761242303770,"stop":1761242303770,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"story","value":"Test Alert Handling"},{"name":"feature","value":"Alerts and Popups"},{"name":"parentSuite","value":"selenium_demoblaze_framework.tests"},{"name":"suite","value":"test_demoblaze_e2e"},{"name":"subSuite","value":"TestDemoBlazeE2E"},{"name":"host","value":"runnervmwhb2z"},{"name":"thread","value":"3461-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"selenium_demoblaze_framework.tests.test_demoblaze_e2e"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"browser_name","value":"'edge'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":3,"skipped":0,"passed":2,"unknown":0,"total":5},"items":[{"uid":"a667483749613b41","reportUrl":"https://Roykunaljit.github.io/selenium-demoblaze-operations/16//#testresult/a667483749613b41","status":"passed","time":{"start":1761253846971,"stop":1761253852551,"duration":5580}},{"uid":"a4203d240cbf83e","reportUrl":"https://Roykunaljit.github.io/selenium-demoblaze-operations/10//#testresult/a4203d240cbf83e","status":"passed","time":{"start":1761249387777,"stop":1761249393679,"duration":5902}},{"uid":"34f917e9b65fb797","reportUrl":"https://Roykunaljit.github.io/selenium-demoblaze-operations/6//#testresult/34f917e9b65fb797","status":"broken","statusDetails":"selenium.common.exceptions.SessionNotCreatedException: Message: session not created: No matching capabilities found\nStacktrace:\n#0 0x55fce4e73fba <unknown>\n#1 0x55fce48f86d0 <unknown>\n#2 0x55fce497ce12 <unknown>\n#3 0x55fce497bf9e <unknown>\n#4 0x55fce497db72 <unknown>\n#5 0x55fce497d946 <unknown>\n#6 0x55fce496fc03 <unknown>\n#7 0x55fce493c7a8 <unknown>\n#8 0x55fce493d421 <unknown>\n#9 0x55fce4e38b28 <unknown>\n#10 0x55fce4e3c87f <unknown>\n#11 0x55fce4e20c49 <unknown>\n#12 0x55fce4e3d405 <unknown>\n#13 0x55fce4e064ff <unknown>\n#14 0x55fce4e61258 <unknown>\n#15 0x55fce4e61432 <unknown>\n#16 0x55fce4e72fa3 <unknown>\n#17 0x7f955d49caa4 <unknown>\n#18 0x7f955d529c6c <unknown>","time":{"start":1761243552854,"stop":1761243552854,"duration":0}},{"uid":"c42d1bea02282e47","reportUrl":"https://Roykunaljit.github.io/selenium-demoblaze-operations/5//#testresult/c42d1bea02282e47","status":"broken","statusDetails":"selenium.common.exceptions.NoSuchDriverException: Message: Unable to obtain driver for MicrosoftEdge using Selenium Manager.; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location","time":{"start":1761243137069,"stop":1761243137069,"duration":0}}]},"tags":[]},"source":"8723fb6f7e58c013.json","parameterValues":["'edge'"]}